lib/ansible/modules/system/ufw.py

```diff
@@ -400,6 +400,8 @@ def ufw_version():
                 execute(cmd + [[command], [value]])
 
         elif command == 'default':
+            if params['direction'] in ['in', 'out', None]:
```

Comment:
Why not have direct checks instead of cross-validation?
Like:
```suggestion
            if params['direction'] not in ['outgoing', 'incoming', 'routed']:
```

Reply:
Very good question. That also makes it more future-proof!